Sources/StreamChatSwiftUI/ChatChannel/ChannelInfo/PinnedMessagesView.swift

```diff
@@ -80,4 +81,11 @@ struct PinnedMessageView: View {
         }
         .padding(.all, 8)
     }
+    
+    private var pinnedMessageSubtitle: String {
+        if message.poll != nil {
```

Comment:
I wonder if it needs `isDeleted` check here as well or it never reaches here for deleted messages.

Reply:
I checked - if it's deleted it's not retrieved by the API, so we're good.